Makefile

```diff
@@ -97,6 +97,7 @@ $(OPENCV_DEPLOY): $(OPENCV_BUILD) virtualenv
 	cp $(OPENCV_BUILD) $(SITE_PACKAGES_DIR)
 
 opencv: $(OPENCV_DEPLOY)
+	$(PIP) install opencv-python
```

Comment:
Should not be required since we actually compile it

Reply:
It's a wrapper not the library itself. I always installed to use ZBarCam and thought it would be nice to install it where opencv is compiled, but I can remove it.

Reply:
Yes I would rather not use it, things seem to work smoothly without it as far as I know